src/core/voteset.rs

```diff
@@ -273,10 +273,27 @@ pub fn verify_tx_version(tx: &Transaction, version: u32) -> bool {
 
 // verify to, tx nonce and valid_until_block
 pub fn verify_tx(tx: &Transaction, height: u64) -> bool {
```

Comment:
I guess it needs to check whether the block version is the same as tx.

Reply:
These two vesion must be equal to current version on chain.
So we don't need to check they are same.

Reply:
And the block version check?

Reply:
yes
https://github.com/cryptape/cita-bft/blob/develop/src/core/cita_bft.rs#L930